### PR TITLE
feat: add document timestamp tests

### PR DESCRIPTION
**Pull request by shuplenkov**
fix: old tests for new dependencies

## Issue being fixed or feature implemented

Need tests for document $createdAt and $updatedAt timestamps

## What was done?

Added test cases for document timestamps:
* fetch created document by time range
* update the created document
* fail to update document with timestamp in violated time frame 
* fail to create a new document with timestamp in violated time frame

## How Has This Been Tested?

Tests

## Breaking Changes

None

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: test/functional/platform/Document.spec.js [hunk 1]
```diff
@@ -142,11 +143,92 @@ describe('Platform', () => {
 
     it('should fetch created documents array', async () => {
```
Comment (shumkov) [inline test/functional/platform/Document.spec.js:144]: Is it outdated?

File: test/functional/platform/Document.spec.js [hunk 2]
```diff
@@ -142,11 +143,92 @@ describe('Platform', () => {
...
+        'customContracts.indexedDocument',
         { where: [['$id', '==', document.getId()]] },
       );
 
       expect(document.toJSON()).to.deep.equal(fetchedDocument.toJSON());
+      expect(fetchedDocument.getUpdatedAt().getTime())
+        .to.be.equal(fetchedDocument.getCreatedAt().getTime());
+    });
+
+    it('should fetch created document by time range', async () => {
```
Comment (shumkov) [inline test/functional/platform/Document.spec.js:155]: should be able to fetch

File: test/functional/platform/Document.spec.js (hunk 1 above)
Comment (shuplenkov) [reply test/functional/platform/Document.spec.js:144]: Why?

File: lib/test/getDAPISeeds.js [hunk 3]
```diff
@@ -1,7 +1,15 @@
...
   return process.env.DAPI_SEED
     .split(',')
-    .map((seed) => ({ service: `${seed}` }));
+    .map((seed) => {
+      const [host, port] = seed.split(':');
+
+      return {
+        host,
+        httpPort: port,
+        grpcPort: port,
```
Comment (shumkov) [inline lib/test/getDAPISeeds.js:10]: They should be different ports.

File: lib/test/getInputsByAddress.js [hunk 4]
```diff
@@ -5,11 +5,11 @@
  * @return {Promise<[]>}
  */
 module.exports = async function getInputsByAddress(dapiClient, address) {
-  const { items: utxos } = await dapiClient.getUTXO(address);
+  const { items: utxos } = await dapiClient.core.getUTXO(address);
   let inputs = [];
 
   if (process.env.NETWORK === 'regtest') {
```
Comment (shumkov) [inline lib/test/getInputsByAddress.js:11]: ”regtest” or ”local”. Please update everywhere.

File: test/functional/platform/Document.spec.js (hunk 1 above)
Comment (shumkov) [reply test/functional/platform/Document.spec.js:144]: It says we fetch document array, but we are fetching just a single doc.

File: .travis.yml [hunk 5]
```diff
@@ -19,7 +19,7 @@ addons:
       - docker-ce
 
 env:
-  - DOCKER_COMPOSE_VERSION=1.25.5
+  - DOCKER_COMPOSE_VERSION=1.25.5 DRIVE_VERSION=0.14.0-dev DAPI_VERSION=0.14.0-dev
```
Comment (shumkov) [inline .travis.yml:22]: We might just forget about these hardcoded versions. Don't we need just update mn-bootstrap?

**Pull request merged**